pyfiglet/__init__.py

```diff
@@ -785,12 +815,23 @@ def main():
                       help='show installed fonts list')
     parser.add_option('-i', '--info_font', action='store_true', default=False,
                       help='show font\'s information, use with -f FONT')
+    parser.add_option('-c', '--color', default=':',
+                      help='''prints text with passed foreground color,
+                            --color=foreground:background
+                            --color=:background\t\t\t # only background
+                            --color=foreground | foreground:\t # only foreground
+                            --color=list\t\t\t # list all colors
+                            COLOR = list[COLOR] | [0-255];[0-255];[0-255] (RGB)''')
```

Comment:
Not sure I understand `list[COLOR]` ?

I'm not sure you need to list the `foreground:`, since `--color=foreground` covers it.

Reply:
For the first one, I wanted to say that must be a color from the list of available colors
The second one, the colon is optional, you can write it or not, so a third party don't have to deal checking if they have to write it the color or not